tracing-core/src/metadata.rs

```diff
@@ -315,13 +316,6 @@ impl<'a> Metadata<'a> {
         self.line
     }
 
-    /// Returns an opaque `Identifier` that uniquely identifies the callsite
-    /// this `Metadata` originated from.
-    #[inline]
-    pub fn callsite(&self) -> callsite::Identifier {
-        self.fields.callsite()
-    }
-
```

Comment:
I still don't understand the purpose of `callsite::Identifier`, like, do we straight up not need it anymore?

Reply:
Yup, pretty sure we don't need it anymore. I'll remove it first thing tomorrow morning.